### DISCUSSION TITLE
Diminuer la puissance de chauffe d'un ACI 2400W à 800W

### DISCUSSION THREAD

**Discussion by lionpeloux** [FredM67/PVRouter-3-phase#91] [Q&A]
Bonjour,

Je lis avec beaucoup d'intérêt les informations proposées sur le repo.
Equipé d'une installation PV récente de 5kWc, je cherche à optimiser mon autoconso via l'usage de mon ballon.
Ce dernier est un ballon **SAUTER ESSENTIEL ACI HYBRID 200L vertical mural**.

* La carte de puissance correspond à la [version kitable](https://github.com/FredM67/PVRouter-3-phase/wiki/Cartes-ACI-monophas%C3%A9e-et-triphas%C3%A9e).
* J'ai consulté le [schéma électrique de cette carte](https://github.com/FredM67/PVRouter-3-phase/blob/main/img/sch%C3%A9ma%20ACI%20mono.jpg) disponible sur le wiki.

**Question :** 

1. Si je déconnecte simplement le fil de phase 5 et/ou le fil de phase 3 pour obtenir une puissance de chauffe réduite (2/3 respectivement 1/3 de 2400W) est-ce que ça fonctionne ?
2. Est-ce que cela risque-t-il de réduire la durée de vie de ma résistance (car seulement une résistance sur les 3 serait utilisée, et donc elle le serait de manière "intensive") ?
3. Question annexe : pourquoi y a t il 2 fils de neutre 1 et 3 ? 1 seul en 2.5mm2 n'aurait il pas suffit ? Est-ce une question de puissance ?!

Je cherche dans un premier temps à aller sur une solution de contrôle tout ou rien (avec horloge ou scénario domotique Home Assitant) plutôt que un solution de routage (avec adaptation de la puissance consommée pour coller au surplus). En réduisant "physiquement" la puissance de chauffe il faudra certes chauffer plus longtemps mais ce sera une puissance (800W) beaucoup plus adaptée à l'autoconsommation (en crête effectif je fais 3.6kW max en juillet).

Merci d'avance pour votre aide !

**Comment by FredM67** [comment]
Bonsoir Lionel,

1. Chaque résistance, au nombre de 3 a priori est reliée sur un des fils rouges. Si tu déconnectes un fil, alors la résistance associée ne sera plus alimentée. Cela fonctionne parfaitement.
2. Non la durée de vie sera la même. Si chaque résistance est prévue pour fonctionner par exemple 10000h (valeur donnée juste pour l'exemple, je n'ai aucune idée de la réalité), alors du point de vue de l'utilisateur, elle claquera 3 fois plus vite car elle fonctionnera 3 fois plus longtemps pour chaque chauffe. Imaginons, le temps de chauffe moyen est de 3h, cela nous fait donc en gros 10000/3=3000 chauffes soit 3000 jours. Si une seule fonctionne, on aura un temps de chauffe de 9h, et donc 10000/9=1000 chauffes soit 1000 jours.
3. Bonne question qu'il faudrait poser à Atlantic ! Potentiellement, c'est pour une histoire de symétrie du connecteur ?!?

Une solution tout-ou-rien ne sera jamais aussi précise qu'un routeur. Un routeur s'adapte en temps réel en fonction de la production et de la consommation de la maison.

Fred

**Comment by lionpeloux** [comment]
Bonjour Fred,

Merci pour ton retour précis et détaillé :

1. ok
2. je crois qu'on dit la même chose. 1 résistance seule s'use trois fois plus vite (car temps de chauffe x3). Mais reste la possibilité de switcher sur une autre résistance quand la première claque (pas pratique ...).

J'ai bien compris l'intérêt du routeur par rapport à cette situation "intermédiaire". Aurais tu une recommandation sur un modèle ? Je penche actuellement sur le MSun-PV, notamment par ce que le forum est très actif. Quel est ton lien avec le mk2pvrouteur ?! Je n'ai pas compris si tu faisais parti des "repreneurs" français ...

Autre question, j'ai l'impression après m'être documenté que c'est préférable de privilégier le hachage par SSR plutôt que via un Triac pour des puissances > 1kW (mon chauffe eau fait 2.4kW). Est-ce que c'est quelque chose que tu confirmes également ?

Merci encore pour tes précieux conseils.

**Comment by FredM67** [comment]
> J'ai bien compris l'intérêt du routeur par rapport à cette situation "intermédiaire". Aurais tu une recommandation sur un modèle ? Je penche actuellement sur le MSun-PV, notamment par ce que le forum est très actif. Quel est ton lien avec le mk2pvrouteur ?! Je n'ai pas compris si tu faisais parti des "repreneurs" français ...

Le mk2pvrouter bien sûr ! Non, je ne fais pas partie des repreneurs, j'étais en étroit contact avec Robin avant qu'il arrête. Tout naturellement, j'ai continué avec les repreneurs, je leur fournis gracieusement l'expertise technique électronique et informatique. 

> Autre question, j'ai l'impression après m'être documenté que c'est préférable de privilégier le hachage par SSR plutôt que via un Triac pour des puissances > 1kW (mon chauffe eau fait 2.4kW). Est-ce que c'est quelque chose que tu confirmes également ?

Techniquement, un SSR et un triac, c'est la même chose. Dans un SSR, les autres composants nécessaires notamment le moc sont encapsulés dans un même boîtier. 
Pour chacun on peut choisir le mode de fonctionnement grâce au type de moc utilisé.
